modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1497,6 +1499,21 @@ private long _getObjectEntryFolderId(
 			}
 		}
 
+		try {
+			ObjectEntryFolder objectEntryFolder =
+				_objectEntryFolderLocalService.
+					getOrAddIncompleteObjectEntryFolder(
+						objectEntryFolderExternalReferenceCode, 0, groupId,
+						companyId, serviceContext);
+
+			return objectEntryFolder.getObjectEntryFolderId();
+		}
+		catch (Exception exception) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(exception);
+			}
+		}
```

Comment:
Since you added this code, I think we should remove the previous one :thinking: 
```
try {
	ObjectEntryFolder objectEntryFolder =
		_objectEntryFolderLocalService.
			getObjectEntryFolderByExternalReferenceCode(
				objectEntryFolderExternalReferenceCode, groupId,
				companyId);
	return objectEntryFolder.getObjectEntryFolderId();
}
catch (Exception exception) {
	if (_log.isDebugEnabled()) {
		_log.debug(exception);
	}
}
```